sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -537,6 +541,7 @@ object Radar {
             override fun onComplete(config: RadarConfig) {
                 locationManager.updateTrackingFromMeta(config?.meta)
                 RadarSettings.setFeatureSettings(context, config?.meta.featureSettings)
+                warmupStandardIntegrityTokenProvider(config)
```

Comment:
This should be probably be gated with `if (fraud)` so we're not calling it for every customer. We could move it into the `if (fraud)` block ~15 lines above

Reply:
Also, is this sync or async?